docs/pages/access-controls/access-request-plugins/ssh-approval-jira.mdx

```diff
@@ -17,178 +38,739 @@ Jira tickets.
+plugin uses TLS credentials. Your organization probably has its own system for
+managing TLS credentials, and that is the system you should use in production.
 
-- Pending
-- Approved
-- Denied
+</Notice>
 
-Teleport's Jira plugin will create a new issue for each new permission request in the first available column on the board. When you drag the request task to the Approved column in Jira, the request will be approved. If you drag the request task to the Denied column in Jira, the request will be denied.
+<Tabs>
+<TabItem label="Executable">
 
-### Get your Jira API token
+### Create a DNS record for the webhook
 
-If you're using Jira Cloud, navigate to [Account Settings → Security → API Tokens](https://id.atlassian.com/manage-profile/security/api-tokens) and create a new app specific API token in your Jira installation. You'll need this token later to configure the plugin.
+Jira requires to access your webhook over the public internet, so you will need
+to assign a domain name to your webhook. Create a DNS `A` record that associates
+your domain name with the IP address of your Jira webhook.
 
-For Jira Server, the URL of the API tokens page will be different depending on your installation.
+### Install Caddy
 
-### Set up Jira webhooks
+Use Caddy to run a webserver to communicate with Let's Encrypt and manage TLS
+credentials. Caddy is an open source webserver with automatic HTTPS provisioning
+functionality. We use it to provide a TLS key and certificate to the Jira
+plugin.
 
-Go to Settings → General → System → Webhooks and create a new webhook for Jira to tell the Teleport plugin about updates.
+Install Caddy on the host where you will run the plugin. For example, run the
+following commands on Debian or Ubuntu hosts:
 
-For the webhook URL, use the URL that you'll run the plugin on. It needs to be a publicly accessible URL (we will show you how to set this up later). Jira requires the webhook listener to run over HTTPS.
+```code
+$ sudo apt install -y debian-keyring debian-archive-keyring apt-transport-https
+$ curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/gpg.key' | sudo gpg --dearmor -o /usr/share/keyrings/caddy-stable-archive-keyring.gpg
+$ curl -1sLf 'https://dl.cloudsmith.io/public/caddy/stable/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/caddy-stable.list
+$ sudo apt update
+$ sudo apt install caddy
+```
 
-The webhook needs to be notified only about new issues being created, issues being updated, or deleted. You can leave all the other boxes empty.
+For other systems, follow the instructions for your system in Caddy's [installation
+guide](https://caddyserver.com/docs/install).
 
-<Admonition
-  type="note"
-  title="Plugin Defaults"
->
-  Jira will send updates about any issues in any projects in your Jira installation to the webhook. We suggest that you use JQL filters to limit which issues are being sent to the plugin.
+### Configure Caddy
```

Comment:
Are we actually using Caddy as a web server or are we just using it to get certs? If we're not using it as a web server, than woudln't something like certbot make more sense?

Reply:
I'm still a little confused. We didn't mention Caddy before, so what has changed that now requires it?

It's not necessarily a bad thing, the barrier to entry gets higher which each extra piece of OSS you have to cobble together to get the system up and running.

Reply:
Caddy itself isn't required, but the user needs some way to provide TLS credentials to the Jira webhook. This requirement hasn't changed since we published the original guide, which [mentions it briefly](https://github.com/gravitational/teleport/blob/5df13390fd57eaf5338dc5cea92f8272fc27cc30/docs/pages/access-controls/access-request-plugins/ssh-approval-jira.mdx#L136-L137) in the configuration section. With this PR, I wanted to help the user remove some decision making in order to get to the objective of the guide more quickly, so I added instructions for getting TLS credentials with `cert-manager` and Caddy. 

I attempted to [provide some context](https://github.com/gravitational/teleport/blob/abcbdcb22ab7812159d3d255061823dc0d1bf020/docs/pages/access-controls/access-request-plugins/ssh-approval-jira.mdx#L90-L91) around the TLS examples, but maybe I could be more explicit about why we're including Caddy/cert-manager instructions?